streams/src/test/java/org/apache/kafka/test/KStreamTestDriver.java

```diff
@@ -147,7 +147,12 @@ protected void after() {
 
     private void initTopology(final ProcessorTopology topology, final List<StateStore> stores) {
         for (final StateStore store : stores) {
-            store.init(context, store);
+            try {
+                store.init(context, store);
+            } catch (final RuntimeException e) {
+                new RuntimeException("Fatal exception initializing store.", e).printStackTrace();
+                throw e;
+            }
```

Comment:
If store.init throws an exception, it fails the tests, but doesn't tell you why. I added this after a fair amount of debugging revealed that this line was throwing. WDYT about leaving it in?

Reply:
The particular exception I ran into was trying to register duplicate metrics, related to the cache name not being split correctly.

Either way; It was handy to have this here, so I wanted to give you the option of leaving it in place.

Reply:
Seems reasonable to me. But this raises a secondary thought will we be keeping `KStreamTestDriver` with the release of our new testing utils?

Reply:
Sounds reasonable to me. Is there a similar issue with `TopologyTestDriver` as well?

Reply:
Possibly. I'll take a look.

Reply:
@bbejeck There is a ticket underway to use TopologyTestDriver in our internal tests (https://issues.apache.org/jira/browse/KAFKA-6474), but it's not clear at this point whether we will actually be able to remove KStreamTestDriver.

Reply:
@guozhangwang , `TopologyTestDriver` uses the same store initialization code path that `KafkaStreams` does, so I'll leave it alone for now. If there's a visibility problem, we'll surely find out.